themes/cp-theme/cp-theme.js

```diff
@@ -14,26 +14,26 @@ import "./custom-style-interface.min.js";
   cpthemeTemplate.setAttribute("style", "display: none;");
   cpthemeTemplate.setAttribute("id", templateId);
   cpthemeTemplate.innerHTML = `<style id="${templateId}-style">:root {
-
+  
```

Comment:
Any idea why this file was changed?  I'm assuming prettier, but prettier should have formatted it when Mark committed it.